mreg/api/v1/serializers.py

```diff
@@ -27,6 +30,49 @@ def validate(self, data):
         return data
 
 
+class CommunitySerializer(serializers.ModelSerializer):
+
+    # Members are all the hosts that have this community assigned.
+    hosts = serializers.SerializerMethodField(read_only=True)
+    global_name = serializers.SerializerMethodField(read_only=True)
+
+    def get_hosts(self, obj):
+        return [host.name for host in obj.hosts.all()]
```

Comment:
No need to create the objects when just seeking a value:
 ```return list(obj.hosts.values_list("name", flat=True))```

Reply:
Thanks! Fixed in https://github.com/unioslo/mreg/pull/572/commits/49a6907c8c16202cea473bd0d383aa930b781399